plugins/continuous-toolbox/README.md

```diff
@@ -1,46 +1,38 @@
-# blockly-plugin-continuous-toolbox [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+# @blockly/continuous-toolbox [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
 
-<!--
-  - TODO: Edit plugin description.
-  -->
-A [Blockly](https://www.npmjs.com/package/blockly) plugin that ...
+A [Blockly](https://www.npmjs.com/package/blockly) plugin that creates a continuous-scrolling style toolbox/flyout that is always open. All of the blocks from all categories are in the flyout together, and the user can either click a category name in the toolbox or scroll to the category they're looking for. This flyout only works as a vertical flyout and it works in both left-to-right and right-to-left modes. Parts of the toolbox style have been changed already, but you can customize the style further by following the [toolbox documentation](https://developers.google.com/blockly/guides/configure/web/toolbox).
 
 ## Installation
 
 ### Yarn
 ```
-yarn add blockly-plugin-continuous-toolbox
+yarn add @blockly/continuous-toolbox
 ```
 
 ### npm
 ```
-npm install blockly-plugin-continuous-toolbox --save
+npm install @blockly/continuous-toolbox --save
 ```
 
 ## Usage
+Simply include both the toolbox and flyout classes from the plugin in the options struct used when injecting Blockly. This style of flyout works best with a toolbox definition that does not use collapsible categories.
+
+Note that this plugin uses APIs introduced in the `3.20200924.3` release of Blockly, so you will need to use at least this version or higher.
 
-<!--
-  - TODO: Update usage.
-  -->
 ```js
 import * as Blockly from 'blockly';
-import {Plugin} from 'blockly-plugin-continuous-toolbox';
+import {ContinuousToolbox, ContinuousFlyout} from '@blockly/continuous-toolbox';
 
 // Inject Blockly.
 const workspace = Blockly.inject('blocklyDiv', {
+  plugins: {
+      'toolbox': ContinuousToolbox,
+      'flyoutsVerticalToolbox': ContinuousFlyout,
+    },
   toolbox: toolboxCategories,
+  // ... your other options here ...
 });
-
-// Initialize plugin.
-const plugin = new Plugin(workspace);
-plugin.init();
 ```
 
-## API
-
-<!--
-  - TODO: describe the API.
-  -->
-
```

Comment:
I think information on `setBlockIsRecyclable` can go here and eventually however we decide someone should be able to limit their flyout width.

Reply:
you're right, but i'll let you add the info for recycling if that's okay? if so you can add the heading back then.

Reply:
Yup, sgtm